styles/prosilver/template/js/scoring_preview.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Is it needed to seperate your JS files? Aren't they loaded every time my JS wiles are loaded?
Every new JS file means for browsers a new query to the server to request the file. This costs time. So seperation should only be done when files are big or they are loaded differently, or to strictly seperate logic (like I have seperated JS functions and onload logic).

So I would suggest to add both your JS files in my onload.js.

You can keep your $(document).ready() around each block, and should comment with a comment above that block why it is there and what it does.